### PR TITLE
Update strings from Kitsune.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@peiying2 could you please review this?

The changes are extensive for two reasons:
* There are changes in the db which triggers a line change about where the string is located.
* Next week we are going to deploy the redesign of the site which also deprecates a lot of templates or moves things around.

This PR will have conflicts due to the commits that are coming from Pontoon. I will rebase it once it's ready to merge.

File: en_US/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2510,49 +2509,43 @@ msgstr ""
...
+#: kitsune/landings/jinja2/landings/get-involved-kb.html:29
 #, fuzzy
 msgid "Improve our help articles"
 msgstr "All Articles"
 
-#: kitsune/landings/jinja2/landings/get-involved-kb.html:33 kitsune/landings/jinja2/landings/get-involved-l10n.html:32 kitsune/landings/jinja2/landings/get-involved-questions.html:33
-#: kitsune/landings/jinja2/landings/mobile/get-involved-kb.html:33 kitsune/landings/jinja2/landings/mobile/get-involved-l10n.html:32
-#: kitsune/landings/jinja2/landings/mobile/get-involved-questions.html:33
+#: kitsune/landings/jinja2/landings/get-involved-kb.html:37
+msgid "1 article can be viewed by 400 million users and translated into 70+ languages"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:2518]: Shoud this be `One article ...` instead of `1 article`

File: en_US/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2619,114 +2610,121 @@ msgstr ""
...
-#: kitsune/landings/jinja2/landings/get-involved-questions.html:28 kitsune/landings/jinja2/landings/mobile/get-involved-questions.html:76
+#: kitsune/landings/jinja2/landings/get-involved-questions.html:30
 #, fuzzy
 msgid "Help users in the forums"
 msgstr "Search the Forum"
 
-#: kitsune/landings/jinja2/landings/get-involved-questions.html:44 kitsune/landings/jinja2/landings/mobile/get-involved-questions.html:45
+#: kitsune/landings/jinja2/landings/get-involved-questions.html:37
+msgid "Solving 1 issue helps up to 1,000 users a day"
+msgstr "Solving 1 issue helps up to 1,000 users a day"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:2620]: same here 1 vs one.

**Comment by akatsoulas** [comment]
Thanks @peiying2 . I updated the wording in the SUMO templates. I will re-extract the strings and update this PR once the review is complete.

File: en_US/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -2955,163 +2953,314 @@ msgstr "You have been unsubscribed from this notification."
...
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:72 kitsune/questions/jinja2/questions/metrics.html:60 kitsune/search/forms.py:92
+#, fuzzy
+msgid "Topics"
+msgstr "Topics:"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:120
+#, fuzzy
+msgid "All articles and threads"
+msgstr "All articles requiring review..."
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:3222]: Should this be a real ellipses vs three periods? What is it not a period? What follows this?

File: en_US/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
...
-msgid "Browse the questions for all the products together in one list."
-msgstr "Browse the questions for all the products together in one list."
+#: kitsune/questions/jinja2/questions/product_list.html:41 kitsune/questions/jinja2/questions/question_list.html:15
+#, fuzzy
+msgid "{product} Support Forum"
+msgstr "Firefox Support"
+
+#: kitsune/questions/jinja2/questions/question_details.html:12
+msgid "no product specified"
+msgstr "no product specified"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:4223]: Is this part of a sentence, with lower case `no`? What proceeds this?

File: en_US/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
...
+msgstr "Replies"
+
+#: kitsune/questions/jinja2/questions/question_details.html:220
+#, python-format
+msgid ""
+"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
+"\">contributors forum</a> requesting that you be white-listed."
+msgstr ""
+"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
+"\">contributors forum</a> requesting that you be white-listed."
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:4329]: The last part of the sentence is grammatically in correct. `If ....", there should be a second part of this sentence.

File: en_US/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
...
+msgstr "I am providing the requested information"
+
+#: kitsune/questions/jinja2/questions/question_details.html:245
+#, fuzzy
+msgid "Needs more information"
+msgstr "Revision {num}"
+
+#: kitsune/questions/jinja2/questions/question_details.html:277
+msgid "You must <a href=\"{login_url}\">log in to your account</a> to reply to posts. Please <a href=\"{ask_url}\">start a new question</a>, if you do not have an account yet."
+msgstr "You must <a href=\"{login_url}\">log in to your account</a> to reply to posts. Please <a href=\"{ask_url}\">start a new question</a>, if you do not have an account yet."
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:4347]: Grammar issue for the `Please ... ` part. remove the comma before `if`. Or if the sentence leads with `If, add a period before `please`.

File: en_US/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
...
+msgstr[0] "<strong>%(recent_replied_count)s tweets</strong> in the last"
+msgstr[1] "<strong>%(recent_replied_count)s tweets</strong> in the last"
 
-#: kitsune/questions/jinja2/questions/email/new_answer.html:22 kitsune/questions/jinja2/questions/question_details.html:462
+#: kitsune/questions/jinja2/questions/question_list.html:78
 #, fuzzy
-msgid "Post a Reply"
-msgstr "Post a reply"
+msgid "0 questions in the last 24 hours have no reply. Good job!"
+msgstr "Found <strong>{n}</strong> result for <strong>{q}</strong> in <strong>{l}</strong>"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:4499]: This one and the one followed (maybe more later) is very hard to tell what the variables are for. Is there a place to put comment somewhere? If not, how do we document this so I can send comms to localizers - this is a workaround but not ideal. Not everyone is on the mailing list, especially new contributors.

File: en_US/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -8680,33 +8641,23 @@ msgstr "Oops, your request was unsuccessful"
...
-msgid "Firefox Accounts"
-msgstr "Firefox"
-
-#: kitsune/sumo/jinja2/handlers/403.html:20
-#, fuzzy
-msgid "SUMO account"
-msgstr "My Account"
+#: kitsune/sumo/jinja2/handlers/403.html:13
+msgid "Please try to log in / sign up first with"
+msgstr "Please try to log in / sign up first with"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:8654]: There might be a few sentences like this, concatenated. I assume it is account, my account, SUMO account. Some may have gender issue not knowing what it is.

File: en_US/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -8718,71 +8669,164 @@ msgstr "An Error Occurred"
...
+msgstr "Tools"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:113
+msgid "Explore resources to help you with your contributions."
+msgstr "Explore resources to help you with your contributions."
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:140 kitsune/sumo/jinja2/includes/common_macros.html:318 kitsune/wiki/jinja2/wiki/document.html:79
+#, fuzzy
+msgid "Download Firefox"
+msgstr "MozillaZine"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:8734]: I assume this will remain in English.

File: en_US/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -8718,71 +8669,164 @@ msgstr "An Error Occurred"
...
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:334 kitsune/wiki/jinja2/wikiparser/hook_refresh_button.html:1
+#, fuzzy
+msgid "Refresh Firefox"
+msgstr "SpreadFirefox"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:338
+#, fuzzy
+msgid "How does it work?"
+msgstr "How to"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:8818]: If this is a UI, should it be How To (title style) or is this followed by a topic?

File: en_US/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -11506,364 +10988,1072 @@ msgstr ""
...
+msgstr "Compare Selected Translations"
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:25
+msgid "Significance - MT: Major Content Changes. M: Content Changes. Empty: Minor Changes."
+msgstr "Significance - MT: Major Content Changes. M: Content Changes. Empty: Minor Changes."
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:25
+msgctxt "First letter of 'Significance'"
+msgid "S"
+msgstr "S"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:11244]: Do you have space to accommodate the translation for Significance, non-Latin scripts will most likely translate it. Same with R in the following string

File: en_US/LC_MESSAGES/djangojs.po [hunk 12]
```diff
@@ -687,23 +687,54 @@ msgid "Community Discussion Only"
 msgstr "Community Discussion Only"
 
 #. {q} is the search query, {l} is the language searched.
-#: kitsune/sumo/static/sumo/tpl/search-results-list.html:42
+#: kitsune/sumo/static/sumo/tpl/search-results-list.html:43
 #, python-format
-msgid "We couldn't find any results for <strong>%(q)s</strong> in <strong>%(l)s</strong>. Maybe one of these articles will be helpful?"
-msgstr "We couldn't find any results for <strong>%(q)s</strong> in <strong>%(l)s</strong>. Maybe one of these articles will be helpful?"
+msgid "Sorry! 0 results found for ‘<span>%(q)s</span>’ in ‘<span>%(l)s</span>’"
+msgstr "Sorry! 0 results found for ‘<span>%(q)s</span>’ in ‘<span>%(l)s</span>’"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/djangojs.po:693]: Need context for variables

File: en_US/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -8516,151 +8470,158 @@ msgstr "The video \"%s\" does not exist."
...
+msgid "Nightly"
+msgstr "Nightly"
 
-#: kitsune/sumo/jinja2/base.html:330 kitsune/sumo/jinja2/base.html:333 kitsune/sumo/jinja2/base.html:354 kitsune/users/jinja2/users/profile.html:79
+#: kitsune/sumo/jinja2/base.html:283
 #, fuzzy
-msgid "Twitter"
-msgstr "Title"
+msgid "Nightly for Android"
+msgstr "Firefox 4 for Mobile"
```
Comment (peiying2) [inline en_US/LC_MESSAGES/django.po:8581]: Is `4` meant to be here?

**Comment by peiying2** [comment]
> Thanks @peiying2 . I updated the wording in the SUMO templates. I will re-extract the strings and update this PR once the review is complete.

I am done for the initial round. Other than specific comments, I summarized a few common issues I found. Some are more important than others.

File: en_US/LC_MESSAGES/django.po (hunk 3 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:3222]: I am not sure where this is coming from. The template has the text in the msgid and the translation should be coming from the `msgstr` so the latter is the "translated" text. This is also marked as fuzzy here so maybe there is an issue with the translation.

File: en_US/LC_MESSAGES/django.po (hunk 9 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:8734]: Probably this needs change on the localization site. It's already marked as fuzzy.

File: en_US/LC_MESSAGES/django.po (hunk 10 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:8818]: Similar here.

File: en_US/LC_MESSAGES/django.po (hunk 13 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:8581]: here too.

File: en_US/LC_MESSAGES/django.po (hunk 9 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:8734]: I believe this needs change int he msgstr string too.

File: en_US/LC_MESSAGES/django.po (hunk 10 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:8818]: similar here.

File: en_US/LC_MESSAGES/django.po (hunk 7 above)
Comment (akatsoulas) [reply en_US/LC_MESSAGES/django.po:4499]: Similar, this should be changed on the translation side.

**Pull request closed without merging**